frontend/src/app/(routes)/staking/components/StakingSidebar.tsx

```diff
@@ -0,0 +1,185 @@
+'use client';
+
+import { Validators } from '@/types/staking';
+import { Tooltip } from '@mui/material';
+import React, { useState } from 'react';
+import DialogAllValidators from './DialogAllValidators';
+import { formatVotingPower, parseBalance } from '@/utils/denom';
+import ValidatorLogo from './ValidatorLogo';
+import { useAppSelector } from '@/custom-hooks/StateHooks';
+import { RootState } from '@/store/store';
+import { formatCoin } from '@/utils/util';
+import StakingStatsCard from './StakingStatsCard';
+
+const StakingSidebar = ({
+  validators,
+  currency,
+  chainID,
+}: {
+  validators: Validators;
+  currency: Currency;
+  chainID: string;
+}) => {
+  const stakedBalance = useAppSelector(
+    (state: RootState) =>
+      state.staking.chains?.[chainID]?.delegations.totalStaked || 0
+  );
+  const totalRewards = useAppSelector(
+    (state: RootState) =>
+      state.distribution.chains?.[chainID]?.delegatorRewards.totalRewards || 0
+  );
+  const tokens = [
+    {
+      amount: stakedBalance.toString(),
+      denom: currency.coinMinimalDenom,
+    },
+  ];
+  const rewardTokens = [
+    {
+      amount: parseInt(totalRewards.toString()).toString(),
+      denom: currency.coinMinimalDenom,
+    },
+  ];
+  return (
+    <div className="staking-sidebar">
+      <div className="flex flex-col gap-6">
+        <div className="flex gap-10">
+          <StakingStatsCard
+            name={'Staked Balance'}
+            value={formatCoin(
+              parseBalance(
+                tokens,
+                currency.coinDecimals,
+                currency.coinMinimalDenom
+              ),
+              currency.coinDenom
+            )}
+          />
+          <StakingStatsCard
+            name={'Rewards'}
+            value={formatCoin(
```

Comment:
formatCoin and parse balance are required, you could have one common function to get the value right.